### DISCUSSION TITLE
Mirrored Shape Aria Chart

### DISCUSSION THREAD

**Discussion by sohanemon** [recharts/recharts#3636] [General]
I want to achieve a chart like 
![this](https://github.com/recharts/recharts/assets/42204040/0bac309e-acf2-41bc-9ada-104cba814866)

I have tried implementing it with the Custom Shape Chart example but failed. Would anyone be able to help me to achieve this?

Thank you so much for your attention and participation.

**Comment by ckifer** [comment]
What does your data look like?
